portfolio.css

```diff
@@ -0,0 +1,102 @@
+* {
+	margin: 0;
+	font-size: 20px;
+	color: #ff4716;
+}
+
+.navbar a {
+	padding: 50% 0 0 5%;
+	text-decoration: none;
+	margin: 50% 0 50% 0;
+	position: relative;
+	top: 30px;
+	font-size: 1.5rem;
+
+	
+}
+
+.navbar {
+	background-color: #feeeeb;
+	padding: 0 0 30px;
+}
+
+.welcome-banner {
+	background-image: url("https://images.pexels.com/photos/2132171/pexels-photo-2132171.jpeg?auto=compress&cs=tinysrgb&dpr=3&h=750&w=1260");
+	width: 100%;
+	height: 100vh;
```

Comment:
height generally doesn't need setting as scrolling vertically is perfectly acceptable (unless you know you need it). Not sure what the `vh` unit is, maybe a typo (or quite likely a thing I don't understand)

Reply:
Vh is a unit I was told to set by Rayna to help with the responsiveness of the welcome-banner